db/migrate/20221122214208_add_user_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
```

Comment:
- [ ] Kindly change user to author according to the project requirement so it will create author_id with each index
` t.references :author, foreign_key: { to_table: 'users' }`

Reply:
@VICTOR-CHUKWU Fixed, thank you! 👍🏼